roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -225,8 +225,8 @@
     component: "{{ kibana_component }}"
     logging_component: kibana
     deploy_name: "{{ kibana_name }}"
-    image: "{{ openshift_logging_image_prefix }}logging-kibana:{{ openshift_logging_image_version }}"
-    proxy_image: "{{ openshift_logging_image_prefix }}logging-auth-proxy:{{ openshift_logging_image_version }}"
+    image: "{{ openshift_logging_kibana_image_prefix }}logging-kibana:{{ openshift_logging_kibana_image_version }}"
+    proxy_image: "{{ openshift_logging_kibana_image_prefix }}logging-auth-proxy:{{ openshift_logging_kibana_image_version }}"
```

Comment:
The proxy image is completely different.  We may need to introduce brand new parameters for the auth-proxy image prefix and version.